cases/515688/pv1.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-one
+spec:
+  capacity:
+    storage: 2Gi
+  accessModes:
+    - ReadWriteOnce
+  nfs:
+    path: /sharezyp
+    server: 10.14.6.144
```

Comment:
It's our nfs server for testing pasted in pad env. For testing usage. It should be only used for openshift-qe I think.